src/lib/duration/constructor.js

```diff
@@ -33,7 +33,10 @@ export function Duration (duration) {
 
     this._locale = getLocale();
 
-    this._bubble();
+    // _bubble returns a new Duration because immutability,
+    // but here in the constructor we need to update `this` with the new values
+    var bubbled = this._bubble();
+    copyDuration(this, bubbled);
```

Comment:
Aaand I got my commits out of order while rebasing things.  The implementation of `copyDuration` appears in the next commit.  (In retrospect, I should've just squashed these two commits into one.)